core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -60,7 +60,7 @@
         <source>Redirect to last page visited</source>
       </trans-unit>
       <trans-unit id="tl_page.redirectBack.1">
-        <source>Redirect the user back to the last page visited instead of the redirect page.</source>
+        <source>Redirect the visitor back to the last page visited instead of the redirect page.</source>
```

Comment:
Why did you not use "member" here? The `redirectBack` field defines where to redirect a **member** after they have logged out.